gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
The generated files have a comment explaining that they were generated at the start. This one doesn't and I wonder if it is supposed to form part of the code base?

Reply:
@sladenSL These (this comment and below) were introduced by your initial gradle change. Do you know if they are significant? In the meantime I will test removing them to see if the build succeeds.

Reply:
Build succeeded with all tests run so appears safe?